Form-Controls/styles.css

```diff
@@ -0,0 +1,60 @@
+
+
+/* header */
+.head{
+    display: flex;
+    justify-content: center;
+}
+/* form */
+input:focus {
+    background-color: rgb(0, 255, 225);
+  }
+
+.form{
+    display: flex;
+    justify-content: center;
+    font-size: 20px;
+ }
+
+/* name */
+.name{
+    display: flex;
+    flex-direction: row;
+    font-size: 20px;
+    margin: auto;
+    padding: 6px;
+}
+
+
+/* email */
+
+.email {
+    font-size: 20px;
+    margin: auto;
+    padding: 6px;
+}
+
+/* colour */
+
+.colour{
+    font-size: 20px;
+    padding: 6px;
+}
+.choosecolour{
+    display: grid;
+    grid-template-rows: 60px 50px;

```

Comment:
Setting  fixed pixel heights like this can cause unexpected problems when a user has large text settings on their browser. If you zoom the page, then everything scales up and looks ok, but if you set the text to a large size, look what can happen:

<img width="1412" alt="Screenshot 2022-11-23 at 14 27 50" src="https://user-images.githubusercontent.com/42203406/203631484-e065857e-9578-4fd1-8657-6413d56dd220.png">


In general, we avoid setting pixel sizes where text is involved, and avoid setting heights on elements (unless the height is a UI standard like 100vh or 56.25%)